filewriter.py

```diff
@@ -210,7 +211,7 @@ def start(self, beam, chan0, navg, nchan, chan_bw, npol, pols, **kwds):
         
         # Create and fill
         self._interface = create_hdf5(self.filename, beam)
```

Comment:
Does this function return an interface handle or the frequencies?

Reply:
It returns the open HDF5 file.